include/materials/material_utility.h

```diff
@@ -0,0 +1,68 @@
+#ifndef MPM_MATERIAL_UTILITY_H_
+#define MPM_MATERIAL_UTILITY_H_
+
+#include <cmath>
+
+#include "Eigen/Dense"
```

Comment:
```suggestion
#include "data_types.h"
```
We can use that as the default include